### PR TITLE
create a standalone script for starting the application in production

### PR DESCRIPTION
**Pull request by afeld**
Just makes it a bit easier to follow and add inline comments to.

### CODE REVIEW

**Comment by bipartisanMas0n** [comment]
FINISHED!

**Comment by bipartisanMas0n** [comment]
jk seriously, what are the details?

File: manifest.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # NOTE: any changes to this file require running `cf push <appname>` directly, rather than `cf-blue-green <appname>`.
 
 # general configuration
-command: bin/rake cf:on_first_instance db:migrate && foreman start -p $PORT
+command: script/sever_start
```
Comment (phirefly) [inline manifest.yml]: misspelling here: `server_start`

**Comment by phirefly** [comment]
Updated with the misspelling fix + :shipit:

**Pull request merged**